add additional installation hint (#66)
moia-oss/eslint-prettier-typescript-config: README.md

   ```sh
   yarn add -D typescript eslint prettier
   yarn add -D @moia-oss/eslint-prettier-typescript-config

>    # it may be needed to also install these packages

Reviewer: ```suggestion
   # in some cases, you may need to install these packages directly:
```
